specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json

```diff
@@ -0,0 +1,680 @@
+          {
+            "$ref": "#/parameters/toParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          },
+          {
+            "$ref": "#/parameters/applyParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Query results.",
+            "schema": {
+              "$ref": "#/definitions/PolicyStatesQueryResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/QueryFailure"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query latest at resource group scope": { "$ref": "./examples/QueryResourceGroupScope.json" }
+        }
+      }
+    },
+    "/{resourceId}/providers/Microsoft.PolicyInsights/policyStates/{policyStatesResource}/queryResults": {
+      "post": {
+        "operationId": "PolicyStates_ListQueryResultsForResource",
+        "description": "Queries policy states for the resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyStatesResourceParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/topParameter"
+          },
+          {
+            "$ref": "#/parameters/orderByParameter"
+          },
+          {
+            "$ref": "#/parameters/selectParameter"
+          },
+          {
+            "$ref": "#/parameters/fromParameter"
+          },
+          {
+            "$ref": "#/parameters/toParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          },
+          {
+            "$ref": "#/parameters/applyParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Query results.",
+            "schema": {
+              "$ref": "#/definitions/PolicyStatesQueryResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/QueryFailure"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query all policy states at resource scope": { "$ref": "./examples/QueryResourceScope.json" },
+          "Query all policy states at subscription level resource scope": { "$ref": "./examples/QuerySubscriptionLevelResourceScope.json" },
+          "Query all policy states at nested resource scope": { "$ref": "./examples/QueryNestedResourceScope.json" },
+          "Query all policy states at subscription level nested resource scope": { "$ref": "./examples/QuerySubscriptionLevelNestedResourceScope.json" }
+        }
+      }
+    },
+    "/providers/Microsoft.PolicyInsights/operations": {
```

Comment:
You dont need this in both the files. This operation is needed per RP namespace.

Reply:
@bulentelmaci Can you fix this? Seems to be a quick fix

Reply:
@sarangan12 Working on it

Reply:
@sarangan12 Pushed changes.